package.json

```diff
@@ -10,7 +10,7 @@
 		"debug"
 	],
 	"license": "public domain",
-	"version": "0.25.0",
+	"version": "0.25.1",
```

Comment:
needs a changelog entry if you update the version here

Reply:
ok, I'll split and and a second one with a Changelog entry